fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -2292,7 +2292,7 @@ public void applyOverdueChargesForLoan(final Long loanId, Collection<OverdueLoan
 
             final JsonElement parsedCommand = this.fromApiJsonHelper.parse(overdueInstallment.toString());
             final JsonCommand command = JsonCommand.from(overdueInstallment.toString(), parsedCommand, this.fromApiJsonHelper, null, null,
-                    null, null, null, loanId, null, null, null, null);
+                    null, null, null, loanId, null, null, null, null,null);
```

Comment:
from static method of JsonCommand class has to be chaned in order to get status value from the JSONCommand.  The change in the signature results in addition of one more parameter value in this class.